docs/reference/command_line.rst

```diff
@@ -19,46 +19,46 @@ Page commands
 
 - Update core routes, from routing files to page manager::
 
-    $ php app/console sonata:page:update-core-routes --site=all
+    bin/console sonata:page:update-core-routes --site=all
 
     You could also remove orphan pages with the ``--clean`` option.
 
-    $ php app/console sonata:page:update-core-routes --site=all --clean
+    bin/console sonata:page:update-core-routes --site=all --clean
 
 - Create snapshots from defined pages::
 
-    $ php app/console sonata:page:create-snapshots --site=all
+    bin/console sonata:page:create-snapshots --site=all
 
 - Cleanup snapshots::
 
-    $ php app/console sonata:page:cleanup-snapshots --site=all --keep-snapshots=5
+    bin/console sonata:page:cleanup-snapshots --site=all --keep-snapshots=5
 
 - Create blocks::
 
-    $ php app/console sonata:page:create-block-container --templateCode=default --blockCode=content_bottom --blockName="Left Content"
+    bin/console sonata:page:create-block-container --templateCode=default --blockCode=content_bottom --blockName="Left Content"
 
 - Clone site with pages::
 
-    $ php app/console sonata:page:clone-site --source-id=1 --dest-id=2 --prefix=Foo
+    bin/console sonata:page:clone-site --source-id=1 --dest-id=2 --prefix=Foo
 
 Please note that you can also give multiple website identifiers to some commands, this way::
 
-    $ php app/console sonata:page:update-core-routes --site=1 --site=2 --site=...
-    $ php app/console sonata:page:create-snapshots --site=1 --site=2 --site=...
-    $ php app/console sonata:page:cleanup-snapshots --site=1 --site=2 --site=...
+    bin/console sonata:page:update-core-routes --site=1 --site=2 --site=...
+    bin/console sonata:page:create-snapshots --site=1 --site=2 --site=...
+    bin/console sonata:page:cleanup-snapshots --site=1 --site=2 --site=...
 
 Debug Commands
 --------------
 
 - Print page composition::
 
-    $ php app/console sonata:page:dump-page sonata.page.cms.snapshot PAGE_ID
-    $ php app/console sonata:page:dump-page sonata.page.cms.snapshot PAGE_ID --extended
+    bin/console sonata:page:dump-page sonata.page.cms.snapshot PAGE_ID
+    bin/console sonata:page:dump-page sonata.page.cms.snapshot PAGE_ID --extended
 
 
 - Render a block::
 
-    $ php app/console sonata:page:render-block sonata.page.cms.snapshot PAGE_ID BLOCK_ID
-    $ php app/console sonata:page:render-block sonata.page.cms.page PAGE_ID BLOCK_ID
+    bin/console sonata:page:render-block sonata.page.cms.snapshot PAGE_ID BLOCK_ID
```

Comment:
Shouldn't this have `.. code-block:: bash`? and all other commands

Reply:
Good catch